### PR TITLE
My planck keymap

### PR DESCRIPTION
**Pull request by jetpacktuxedo**
based on a blend of my minivan keymap and the planck default keymap

### CODE REVIEW

File: keyboards/planck/keymaps/jetpacktuxedo/config.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/planck/keymaps/jetpacktuxedo/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/planck/keymaps/jetpacktuxedo/config.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/jetpacktuxedo/config.h]: This line is not actually needed, and should be removed. 

Specifically, this is already included by the keyboard's config.h file, which is included by the build script.

File: keyboards/planck/keymaps/jetpacktuxedo/keymap.c [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead?

File: keyboards/planck/keymaps/jetpacktuxedo/keymap.c [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+#include "action_layer.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: Could you remove this include?  

It's actually included already, so it's redundant here. 
Though, the muse.h is fine.

File: keyboards/planck/keymaps/jetpacktuxedo/keymap.c [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+  float plover_song[][2]     = SONG(PLOVER_SOUND);
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
```
Comment (drashna) [inline keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: (Optional)  If you're only using the QWERTY default layer, and no other layers, then you may want to remove this, as it's not needed.   

Same with the default layer songs in your config.h

**Comment by jetpacktuxedo** [comment]
All of the things that you commented were pulled directly from the planck default keymap :joy: 

I'll clean em up anyway though. Might take a day or two as I'm out of town at the moment.

**Comment by drashna** [comment]
Well .... "do as I say, not as I do" .... 🤣

File: keyboards/planck/keymaps/jetpacktuxedo/keymap.c (hunk 3 above)
Comment (jetpacktuxedo) [reply keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: Hmmm... so I got the other fixes you asked for, but this one gets me
```
Compiling: keyboards/planck/keymaps/jetpacktuxedo/keymap.c                                         keyboards/planck/keymaps/jetpacktuxedo/keymap.c:17:26: fatal error: QMK_KEYBOARD_H: No such file or directory
 #include "QMK_KEYBOARD_H"
                          ^
compilation terminated.
```

Comment (drashna) [reply keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: It shouldn't be in quotes.  it should be: `#include QMK_KEYBOARD_H`

Comment (jetpacktuxedo) [reply keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: Oh, yep, that did it! Misled by my syntax highlighting...

File: keyboards/planck/keymaps/jetpacktuxedo/keymap.c (hunk 5 above)
Comment (jetpacktuxedo) [reply keyboards/planck/keymaps/jetpacktuxedo/keymap.c]: I want to leave plover in there so I can start to learn it :)

Fixing the rest though.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**